sentry-android-replay/src/main/java/io/sentry/android/replay/ReplayCache.kt

```diff
@@ -0,0 +1,167 @@
+package io.sentry.android.replay
+
+import android.graphics.Bitmap
+import android.graphics.Bitmap.CompressFormat.JPEG
+import android.graphics.BitmapFactory
+import io.sentry.SentryLevel.DEBUG
+import io.sentry.SentryLevel.ERROR
+import io.sentry.SentryLevel.WARNING
+import io.sentry.SentryOptions
+import io.sentry.android.replay.video.MuxerConfig
+import io.sentry.android.replay.video.SimpleVideoEncoder
+import io.sentry.protocol.SentryId
+import java.io.Closeable
+import java.io.File
+
+internal class ReplayCache(
```

Comment:
This looks like a good thing to expose to Flutter. My idea is 
- crate a `ReplayCache` from a Flutter kotlin binding code
- create screenshots in Flutter
- write bitmaps to the cache directory directly in Flutter
- notify ReplayCache by calling `addFrame(filename)`
- call `createVideoOf()` to get the video file path
- send the replay envelope from Flutter